http/tokenV2/authorized_keys.go

```diff
@@ -0,0 +1,97 @@
+package tokenV2
+
+import (
+	b64 "encoding/base64"
+	"fmt"
+	"strings"
+
+	"golang.org/x/crypto/ssh"
+
+	"github.com/lestrrat-go/jwx/jwk"
+)
+
+// authorizedKey is an SSH authorized key
+type authorizedKey struct {
+        Key ssh.PublicKey
+        Comment string
+        Options []string
+        JWK jwk.Key     
+}               
+
+// String returns a string representation of an authorized key 
+func (a authorizedKey) String() string {
+        return fmt.Sprintf("%v %v %v %v", a.Key.Type(), b64.RawStdEncoding.EncodeToString(a.Key.Marshal()), a.Comment, a.Options)         
+}  
+
+// jwkFromSSHKey converts a standard SSH library key to a JWX jwk.Key type
+func jwkFromSSHKey(key ssh.PublicKey) (jwk.Key, error) {
+	// Ensure the provided key implements the optional ssh.CryptoPublicKey interface, which
+	// is able to return standard go crypto primitives. These primitives are needed to convert
+	// the key into a JWX jwk key.
+	var standardKey interface{}
+	if cryptoPublicKey, ok := key.(ssh.CryptoPublicKey); ok {
+		// Convert the ssh.PublicKey type to a go standard library crypto type (of unknown/interface{} type).
+		standardKey = cryptoPublicKey.CryptoPublicKey()
+	} else {
+		return nil, fmt.Errorf("key %v does not implement the ssh.CryptoPublicKey interface and cannot be converted")
```

Comment:
It's not a programmer's error. Sysadmins could type an entry in the authorized_keys file that ends up loading a key type that doesn't implement this. We really do need to protect against it to avoid a potentially confusing panic event.

Reply:
could use `%T`